templates/azure/sql.html

```diff
@@ -46,64 +46,117 @@ <h1>SQL Server on Ubuntu Pro for Azure</h1>
   <section class="p-section">
     <div class="u-fixed-width p-section--shallow">
       <hr />
-      <h2>What’s inside SQL Server on Ubuntu Pro?</h2>
+      <h2>What's inside SQL Server on Ubuntu Pro?</h2>
```

Comment:
No content changes here, just a different implementation for this custom list which is more accessible (see pr description)